### PR TITLE
Azure NetApp Files Backup scripts

### PR DESCRIPTION
**Pull request by TMEsaranravi**
These scripts are for backing up Azure NetApp files as Generic NAS. Added two scripts. One for creating a snapshot and the second one for deleting a snapshot.

### CODE REVIEW

File: adapters/NAS/AzureNetAppFiles/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Overview
+
+Cohesity custom pre & post script for Azure NetApp Files, allows you to backup specific snapshots of Azure NetApp Files to Cohesity storage.
+
+### Prerequisites
+
+Below is the list of prerequisites which should meet before configuring the Protection Group:
+
+1. Ensure that you have a Linux Control VM configured and have access to Azure NetApp Files volumes
```
Comment (pyashish) [inline adapters/NAS/AzureNetAppFiles/README.md]: It'd be great to add dependencies and how to install them , for example Get-AzNetAppFilesSnapshot  can be got from installing netapps powershell cmdlets right ? probably give powershell gallery link to it.

Comment (pyashish) [reply adapters/NAS/AzureNetAppFiles/README.md]: I see this is using powershell core, min supported version and tested version could also be added

File: adapters/NAS/AzureNetAppFiles/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  ``` bash
+  # Copy create_anf_snapshot.ps1 content and paste in the file create_anf_snapshot.ps1
+
+cat > create_anf_snapshot.ps1
+
+# Press Contrl+D to save the file
+
+   # Copy delete_anf_snapshot.ps1 content and paste in the file delete_anf_snapshot.ps1
+
+   cat > delete_anf_snapshot.ps1
```
Comment (pyashish) [inline adapters/NAS/AzureNetAppFiles/README.md]: I'd just say copy the file from the repository to the powershell VM

File: adapters/NAS/AzureNetAppFiles/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+```
+
+### Run script
+
+* The script will now run whenever the protection group is run
+
+### Have any question
+
+Send me an email at saran.ravi@cohesity.com
```
Comment (pyashish) [inline adapters/NAS/AzureNetAppFiles/README.md:97]: Is there an alias to reach ? this will be spof

**Review by pyashish — COMMENTED** [review]
Great commit , minor nits

File: adapters/NAS/AzureNetAppFiles/README.md (hunk 1 above)
Comment (TMEsaranravi) [reply adapters/NAS/AzureNetAppFiles/README.md]: done

File: adapters/NAS/AzureNetAppFiles/README.md (hunk 2 above)
Comment (TMEsaranravi) [reply adapters/NAS/AzureNetAppFiles/README.md]: done

File: adapters/NAS/AzureNetAppFiles/README.md (hunk 3 above)
Comment (TMEsaranravi) [reply adapters/NAS/AzureNetAppFiles/README.md:97]: I am still getting the alias from the team. Will do another PR once I have the info. I also have a deployment guide in progress and will include that link as well in the readme file.

**Review by pyashish — APPROVED** [review]
Congratulations on first contribution, hope many more to come!!!!

**Pull request merged**